src/libpmemfile/path_resolve.c

```diff
@@ -110,7 +110,7 @@ resolve_symlink(struct resolved_path *result,
 		}
 	}
 
-	if (! *is_last_component)
+	if (!(*is_last_component))
```

Comment:
In that case, please responde to Sarah's comment about it in:
https://github.com/pmem/pmemfile/pull/95